### PR TITLE
add 5 minute videos

### PR DESCRIPTION
**Pull request by alex-fusionauth**

### CODE REVIEW

**Review by mark-robustelli — APPROVED** [review]
Looks good.

File: astro/src/css/style.css [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 lite-youtube{
   @apply aspect-video;
-  max-width: none;
+  max-width: none !important;
```
Comment (lyleschemmerling) [inline astro/src/css/style.css:11]: why is this needed?

**Comment by lyleschemmerling** [comment]
Not specific to this PR but the 5-minute feedback survey component completely busts the layout on mobile.

Also, all three of those quickstarts are using the dotnet image for their icon

File: astro/src/css/style.css (hunk 1 above)
Comment (alex-fusionauth) [reply astro/src/css/style.css:11]: https://inversoft.slack.com/archives/C02CR2M6SRK/p1697738837291899

**Comment by alex-fusionauth** [comment]
> Not specific to this PR but the 5-minute feedback survey component completely busts the layout on mobile.
> 
> Also, all three of those quickstarts are using the dotnet image for their icon

I believe @SeanBobby is already addressing this on another issue.

File: astro/src/css/style.css (hunk 1 above)
Comment (lyleschemmerling) [reply astro/src/css/style.css:11]: Alright the tailwind class for this is `max-w-full` which would be preferable to using `!important`

**Comment by SeanBobby** [comment]
Icons have been designed and push to dev. @mooreds @lyleschemmerling feel free to push to master when you have a chance.
